manifests/a/AngeloConvento/OverKeys/0.2.5/AngeloConvento.OverKeys.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: AngeloConvento.OverKeys
+PackageVersion: 0.2.5
+InstallerType: inno
+InstallerSwitches:
+  Silent: "/VERYSILENT"
+  SilentWithProgress: "/SILENT"
+Installers:
+  - Architecture: x64
+    Scope: machine
+    InstallerUrl: https://github.com/conventoangelo/OverKeys/releases/download/v0.2.5/overkeys_0.2.5_x64_setup.exe
+    InstallerSha256: 856D4C78EA7D8799A5123BB85839DEEEE704DE26D78892C78E6E836DEC4962D2
+    InstallerSwitches:
+      Custom: /ALLUSERS
+  - Architecture: x64
+    Scope: user
+    InstallerUrl: https://github.com/conventoangelo/OverKeys/releases/download/v0.2.5/overkeys_0.2.5_x64_setup.exe
+    InstallerSha256: 856D4C78EA7D8799A5123BB85839DEEEE704DE26D78892C78E6E836DEC4962D2
+    InstallerSwitches:
+      Custom: /CURRENT

```

```suggestion
      Custom: /CURRENTUSER
```
`/CURRENT` still shows the dialogue